### DISCUSSION TITLE
Error while trying to install radmc3dPy.

### DISCUSSION THREAD

**Discussion by cyber-prags** [dullemond/radmc3d-2.0#30] [General]
I am facing an error "Fast (Fortran90) Mie-scattering module could not be imported. Falling back to the slower Python version" while trying to import radmc3dPy. Cam anyone help me out to resolve this?
![Screenshot from 2023-01-24 00-36-14](https://user-images.githubusercontent.com/74003758/214127857-39eb11d6-262f-4191-bc68-9f2a04aea4cd.png)

**Comment by dullemond** [comment]
Hi,

If the f90 Mie scattering stuff does not compile, that's not a problem. The code is nevertheless usable. There is a slower Python-based version of the classic Mie code. But I recommend a MUCH better option: If you want to make opacities, use Carsten Dominik's `optool` code. It's awesome! 

[https://github.com/cdominik/optool](https://github.com/cdominik/optool)

and it outputs opacity files in the format you need for RADMC-3D. 

Best wishes,
Kees
